aspnetcore/web-api/define-controller/samples/WebApiSample.Api.21/Controllers/MyBaseController.cs

```diff
@@ -1,10 +1,10 @@
-﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc;
 
 namespace WebApiSample.Api._21.Controllers
 {
     #region snippet_ControllerSignature
     [ApiController]
-    public class MyBaseController
+    public abstract class MyBaseApiController: ControllerBase
```

Comment:
nit: Common standard with abstract classes is to have Base as the suffix.

```suggestion
    public abstract class MyApiControllerBase : ControllerBase
```